lib/browser.ts

```diff
@@ -910,7 +910,7 @@ export class ProtractorBrowser extends Webdriver {
       logger.info('  e.g., list(by.binding(\'\')) gets all bindings.');
       logger.info();
     };
-    this.debugHelper.init(debuggerClientPath, onStartFn, opt_debugPort);
+    this.debugHelper.init(debuggerClientPath, true, onStartFn, opt_debugPort);
```

Comment:
Renamed into two different functions on the theory that the best documentation is good naming.